Sources/Operation.swift

```diff
@@ -36,11 +49,17 @@ struct OperationBuilder: Builder {
     let responses: [Int : Reference<ResponseBuilder>]
     let defaultResponse: Reference<ResponseBuilder>?
     let deprecated: Bool
+    let operationId: String?
+    let tags: [String]?
+    let security: [SecurityRequirement]?
 
     init(map: Map) throws {
         summary = try? map.value("summary")
         description = try? map.value("description")
         parameters = (try? map.value("parameters")) ?? []
+        operationId = try? map.value("operationId")
+        tags = try? map.value("tags")
+        security = try? map.value("security")
```

Comment:
If non optional you can do:
tags = (try? map.value("tags")) ?? []
security = (try? map.value("security")) ?? []